src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -39,6 +39,12 @@ object WebReportRouteConfig : RouteConfig
                     "/run-report/",
                     ReportController::class, "getRunReport")
                     .secure(runReports),
+            WebEndpoint(
```

Comment:
I think this could be confusable with the list of reports which a user can view/review, as opposed to their runnable reports. Could just be `/reports/runnable`? But maybe we should be giving more thought to how we organise these new routes for the features in this sprint. At the least, I think the tickets around 'my running report logs' should probably go under a separate route.

Reply:
Agreed - have made it `/reports/runnable` for now, but rationalising then (possibly with a new controller?) can be done depending on direction we take with other tickets in this sprint.